httprange/httprange.go

```diff
@@ -0,0 +1,159 @@
+// Package httprange provides RFC7233 range parsing.
+package httprange
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+)
+
+// Bytes is the bytes range type specified in the RFC.
```

Comment:
NIT: Maybe some very minimal documentation for people who don't want to read the whole RFC?